### ISSUE TITLE
2 WB solutions, same damage, different Hero orders

### ISSUE THREAD

**Issue by Vitae666** [Alya-N/C-Hero-Calc#55]
Solution for [Followers:       0 | bornag:1 <==]:
[Followers:       0 | spike:66 b-day:99.2 emily:99 **masterlee:99.2 raiderrose:99.6** aurora:99.6 <==]
Boss Damage Done: 7,821

Solution for [Followers:       0 | bornag:1 <==]:
[Followers:       0 | spike:66 b-day:99.2 emily:99 **raiderrose:99.6 masterlee:99.2** aurora:99.6 <==]
Boss Damage Done: 7,821

Now, granted, the 1st one was from a month or so ago, and the 2nd was just now.
So, unless you made some change(s) with those 2 heroes that I don't remember, thought I'd mention it. Only because it seems weird to me that they would do the exact same damage :grinning:

**Comment by Alya-N** [comment]
Well, since it's Bornag, of course they deal the same damage, but as to why they are different, the only reason I've seen is that the heroes were put into the calc in a different order. Shouldn't happen in theory if you're using macrocreator though. I'll try checking it, but can't promise I'll find anything.

**Comment by Vitae666** [comment]
Huh...Guess I forgot how Bornag works. Was thinking it was like the others. My bad.
As to order, dunno. Didn't think that would matter, but since I was wrong about Bornag, I guess I can see how that the order makes a difference. 
I'm assuming that order placed in the calc does NOT apply to the other Bosses?

**Comment by Alya-N** [comment]
Well, probably not to bosses (unless two lineups with the same damage exist), but it could affect which solution comes up first for DQs.

**Comment by Vitae666** [comment]
Meh, if it beats the DQ, who cares :)
I'll try a test run at some point on the bosses using just the ones that the calc says was the best. I'll use diff orders a few times and see what happens.

**Comment by Vitae666** [comment]
Ok, so, here's what I've got. I've attached my files.
The text files are the longer copy/paste of the calc output. The Excel is just the important parts, but also highlights the results are the same.
In a nut shell between A-Z, Z-A and Random: 
LOC, KRY and BOR would have some variations between the lineups but have the **same damage output** listed.
I ran them multiple times as tests because I never believe the 1st result, and each time, the results were the exact same.
LOC: Z-A & Random would be the same lineup with _A-Z_ being different
KRY: A-Z & Z-A would be the same lineup with _Random_ being different
BOR: A-Z & Random would be the same lineup with _Z-A_ being different
MOAK & DOY would never waver. Always the same lineup

I guess the question is, is this accurate? What I mean is, if it's displayed in the Calc, does it actually DO the same damage with the different lineups?

[A-Z.txt](https://github.com/Alya-N/C-Hero-Calc/files/5025332/A-Z.txt)
[Z-A.txt](https://github.com/Alya-N/C-Hero-Calc/files/5025331/Z-A.txt)
[Random.txt](https://github.com/Alya-N/C-Hero-Calc/files/5025330/Random.txt)
[Findings.xlsx](https://github.com/Alya-N/C-Hero-Calc/files/5025369/Findings.xlsx)

EDIT:
Ok, so ran the Battle Replays for LOC/KRY/BOR and yeap, the total damage was always the same with the different lineups.
So, guess it's all good. WEIRD...but good... :grin:
